app/src/main/java/com/vero/tasky/agenda/data/repository/EventRepositoryImpl.kt

```diff
@@ -18,48 +21,30 @@ class EventRepositoryImpl(
     private val api: EventApi,
     private val eventDao: EventDao,
     private val modifiedAgendaItemDao: ModifiedAgendaItemDao,
-    private val multipartParser: MultipartParser
+    private val multipartParser: MultipartParser,
+    private val createEventWorkerRunner: CreateEventWorkerRunner,
```

Comment:
Personally to avoid creating 2 similar workers (one for Create, oke for Update) I just used the same worker, but changed the api destination based on if it was create or update

In that way, all stuff related to it, like the multiPartParser, can be added to the worker and not to repository

Reply:
I still need multipartParser here to count how many images will be skipped